packages/carbon-web-components/src/components/breadcrumb/breadcrumb-overflow-menu.ts

```diff
@@ -18,12 +18,13 @@ import { carbonElement as customElement } from '../../globals/decorators/carbon-
  * Overflow menu in breadcrumb.
  *
  * @element cds-breadcrumb-overflow-menu
+ * @csspart icon - The icon. Usage: `cds-breadcrumb-overflow-menu::part(icon)`
  */
 @customElement(`${prefix}-breadcrumb-overflow-menu`)
 class CDSBreadcrumbOverflowMenu extends CDSOverflowMenu {
   render() {
     return html`
-      <slot name="icon">
+      <slot name="icon" part="icon">
```

Comment:
We don't need a `part` attribute on `<slot>` tags. Anything slotted into here is part of the light DOM, which users of the component have easy access already for styling.